R/dsl.R

```diff
@@ -155,7 +159,9 @@ do_package_checks <- function(...,
 #' @inheritParams step_build_pkgdown
 #' @inheritParams step_setup_push_deploy
 #' @inheritParams step_do_push_deploy
-#' @param build_only Build the pkgdown site but do not deploy it.
+#' @param build_only `[flag]`\cr Build the pkgdown site but do not deploy it. Removes step
```

Comment:
I wonder if a `deploy` argument makes more sense here -- would be the inverse of `build_only` . Perhaps easier to describe/document?

Reply:
> I wonder if a deploy argument makes more sense here -- would be the inverse of build_only . Perhaps easier to describe/document?

Hard to say, probably personal taste. Feel free to change it, I am unbiased here :)

Reply:
I'll tweak.